lerna.json

```diff
@@ -0,0 +1,11 @@
+{
+  "lerna": "2.4.0",
+  "version": "1.0.2",
```

Comment:
You definded version of your extension like 0.0.1: https://github.com/eclipse/che-theia-factory-extension/pull/1/files#diff-2c9e1ab7353b6750c47de1b55b578910R6, so I think here should be the same version